src/service/db/short_url.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2024 Zinc Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::sync::Arc;
+
+use anyhow::anyhow;
+use config::utils::json;
+use infra::db::Event;
+
+use crate::{
+    common::infra::config::URL_MAP,
+    service::db::{self},
+};
+
+// DBKey to set short URL's
+pub const SHORT_URL_KEY: &str = "/short_urls/";
+
+pub async fn get(short_id: &str) -> Result<String, anyhow::Error> {
+    match URL_MAP.get(short_id) {
+        Some(val) => Ok(val.to_string()),
+        None => {
+            let val = db::get(&format!("{SHORT_URL_KEY}{short_id}"))
```

Comment:
Here, there is no `db::get` for your key. you should call your infra db model `infra::short_urls::get(xxx)`

Reply:
Fixed